src/pages/index.js

```diff
@@ -96,15 +165,57 @@ addCardPopup.setEventListener();
 //addCardForm.addEventListener("submit", handleAddCardFormSubmit);
 profileAddButton.addEventListener("click", () => {
   addCardPopup.open();
-})
+});
+/*-------------------------CARD deletion -----------*/
+const deleteCardPopup= new PopupWithConfirmation({
+  popupSelector:"#delete-card-modal",
+  handleFormSubmit:(cardId, cardElement) => {
+   // console.log(`Deleting card with cardId=${cardId}`);
+    // Proceed to delete card from server
+    deleteCardPopup.setLoadingState(true);
+    api
+      .deleteCard(cardId)
+      .then(() => {
+        cardElement.remove();
+        deleteCardPopup.close();
+      })
+      .catch((err) => {
+        console.error("Delete card error:", err);
+      })
+      .finally(() => {
+        // Reset the loading state regardless of success or error
+        deleteCardPopup.setLoadingState(false);
+      });
+  },
+});
+deleteCardPopup.setEventListener();
+
 /*------------------------- Section-------------------------- */
 const cardSection = new Section({
-  items: initialCards,
   renderer: renderCard,
 },
   ".gallery__list");
+//Get user user cards
 
+const api = new Api({
+  headers: {
+    authorization: "1a37d956-9fa4-4c51-a36f-94e001ed1e8f",
+    "Content-Type": "application/json"
+  }
+});
+
+api.getInitialCards()
+  .then(cards => {
+    cardSection.renderItems(cards);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
+  api.getUserInfo()
+  .then(data=>{
+    profileInfo.setUserInfo(data.name,data.about,data.avatar);
+  })
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**